src/Resources/public/js/startup.js

```diff
@@ -319,7 +333,9 @@ pimcore.plugin.customermanagementframework = Class.create(pimcore.plugin.admin,
         this.addSegmentAssignmentTab(document, 'document', type);
     },
 
-    postOpenAsset: function (asset, type) {
+    postOpenAsset: function (event) {
+        var asset = event.detail.object;
```

Comment:
```suggestion
        var asset = event.detail.asset;
```
https://github.com/pimcore/pimcore/blob/11.x/bundles/AdminBundle/public/js/pimcore/events.js#L35